src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -156,21 +157,28 @@ public static boolean shouldBeHearingsInWelshFlag() {
 
     public static List<HearingLocations> getHearingLocations(CaseManagementLocation caseManagementLocation) {
         HearingLocations hearingLocations = new HearingLocations();
-        hearingLocations.setLocationId(caseManagementLocation.getBaseLocation());
-        hearingLocations.setLocationType("court");
+        List<HearingLocations> location;
```

Comment:
(minor) should be locations

Reply:
Might be minor, but very important for readability - good spot Del ;)

Reply:
Thank you